Src/FluentAssertions/Common/Services.cs

```diff
@@ -46,6 +46,6 @@ public static void ResetToDefaults()
 #else
         ConfigurationStore = new NullConfigurationStore();
 #endif
-        ThrowException = TestFrameworkProvider.Throw;
+        ThrowException = new TestFrameworkProvider(Configuration).Throw;
```

Comment:
@jnyrup Did you think about the outcome of creating the provide with every thrown exception?
The class was "designed" before for e.g. one-time detection of the framework.

Reply:
This line is only invoked once from the static constructor of `Services`, unless you explicitly class `Services.ResetToDefaults()`.